app/views/content/events/get-the-most-from-events/_steps.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="row">
+<div class="row" style="margin-top:60px">
```

Comment:
move inline styles to an scss file

Reply:
thanks, this was a temporary fix. is there already a css rule for margins that I can use instead?

Reply:
I think either add them to `app/webpacker/styles/sections/hero.scss` or create a new file, e.g. `app/webpacker/styles/components/landing-header.scss` with any specific styles you need in there

Reply:
this wasn't to do with the hero/header, it was part of the content of the page. .col-space-l (layout.scss) puts margin on the bottom. wondered if there was one for top?

Reply:
I don't think there is one, but feel free to define a new style in `app/webpacker/styles/layout.scss`

Reply:
done, added style to layout.scss and updated file